### PR TITLE
Generate instructions.html from README.md.

### PR DESCRIPTION
**Pull request by mattwhisenhunt**
@garrettjonesgoogle PTAL

### CODE REVIEW

File: instructions.html [hunk 1]
```diff
@@ -1,72 +1,198 @@
 <html>
 <title>Setup Instructions for the Google API Client Library for Java</title>
 <body>
-<h2>Setup Instructions for the Google API Client Library for Java</h2>
+<h1>Google APIs Client Library for Java</h1>
```
Comment (garrettjonesgoogle) [inline instructions.html]: The readme.md content doesn't really seem like a good fit for an instructions page...

**Comment by garrettjonesgoogle** [comment]
but...

**Comment by mattwhisenhunt** [comment]
I thought there was an open issue, there is now #1072.

**Comment by garrettjonesgoogle** [comment]
Well my concern was that this was merged...

**Comment by mattwhisenhunt** [comment]
I have updated the PR name to reflect the contents of a new forced push that left instructions.html untouched.

**Comment by garrettjonesgoogle** [comment]
Ahh ok.

**Pull request merged**